### PR TITLE
Community navigation

### PR DESCRIPTION
**Pull request by silverailscolo**
Clean up of html, add sidebars to clinics etc pages, remove old 2020 web parts, update CodeFlower for 2022

### CODE REVIEW

File: community/clinics/NMRA2008/index.shtml [hunk 1]
```diff
@@ -16,8 +16,9 @@
 
 <hr class="hide">
 
-<div class="no-sidebar" id="mBody">
-	<div id="mainContent">
+<div id="mBody">
+    <!--#include virtual="../Sidebar.shtml" -->
+    <div id="mainContent">
```
Comment (jerryg2003) [inline community/clinics/NMRA2008/index.shtml:21]: Should line 20 be referencing SidebarCommunity.shtml?

Comment (silverailscolo) [reply community/clinics/NMRA2008/index.shtml:21]: ../Sidebar contains an include for that, plus one for clinic details. So I don't think so

File: community/examples/Gallery.shtml [hunk 2]
```diff
@@ -36,18 +39,15 @@ in the Fall 2016 issue of the
...
--->
-<a href="images/5519763-11612079-thumbnail.gif"><img src="images/5519763-11612079-thumbnail.gif" class="floatRight" width="200" height="135" alt="OWN panel" /></a>
-
+<p>
 Kent Williams'
-<a href="http://ownry.squarespace.com">Oregon Washington Navigation &amp; Railway</a>
-(now dismantled) used PanelPro for it's signal controls and
-<a href="http://ownry.squarespace.com/blog/2011/4/7/ctc-dispatcher-panel-on-the-ownry.html?SSScrollPosition=0">dispatcher panel</a>.
+<a href="https://web.archive.org/web/20160818193334/http://ownry.squarespace.com/">Oregon Washington Navigation &amp; Railway</a>
+(now dismantled) used PanelPro for its signal controls and
```
Comment (jerryg2003) [inline community/examples/Gallery.shtml:45]: The external inks on lines 44 and 47 appear not to work.  Noted in Issue #395.  May not be possible to find...

File: community/examples/Gallery.shtml [hunk 3]
```diff
@@ -59,19 +59,18 @@ Well over a decade ago, Nick Kulp already created a program to drive his
...
-
-Bob Bucklew built a realistic CTC panel for his
-Quaker Valley Railroad.
+<p>
+Bob Bucklew built a realistic CTC panel for his Quaker Valley Railroad.
 He's created a very nice series of web pages on
-<a href="http://www.quaker-valley.com/CTC/QV_CTCnew.html">how it's done</a>.
+<a href="http://www.quaker-valley.com/CTC/QV_CTCnew.html">how it was done</a>.<br>
+<a href="http://www.quaker-valley.com/CTC/JMRI_QV_CTC.gif">
+    <img src="http://www.quaker-valley.com/CTC/JMRI_QV_CTC.gif" width="256" height="186" alt="QV panel" />
```
Comment (jerryg2003) [inline community/examples/Gallery.shtml:67]: The gif doesn't display on my browser (Chrome under Windows) although the actual links appear to work.

**Review by jerryg2003 — APPROVED** [review]

File: community/examples/Gallery.shtml (hunk 2 above)
Comment (silverailscolo) [reply community/examples/Gallery.shtml:45]: The links open with a delay, but they open on Safari Mac

**Pull request merged**